src/Symfony/Component/Routing/Generator/CompiledUrlGenerator.php

```diff
@@ -20,15 +20,14 @@
  */
 class CompiledUrlGenerator extends UrlGenerator
 {
-    private array $compiledRoutes = [];
```

Comment:
We lose the initializer here. Reminder: This is an issue for child classes that skip the parent constructor.

Is this acceptable in this case?

Reply:
right, thanks ! I have added back the initialization to the promoted property

Reply:
this is not added back. What you added is a default value for the argument, not an initializer for the property (and by doing that, you changed the constructor signature)